lib/audit/catalog_to_moab.rb

```diff
@@ -66,32 +66,33 @@ def check_catalog_version
     # TODO: anything special if preserved_copy.status is not OK_STATUS? - see #480
 
     unless online_moab_found?(druid, storage_dir)
+      update_status(PreservedCopy::ONLINE_MOAB_NOT_FOUND_STATUS)
       results.add_result(PreservedObjectHandlerResults::ONLINE_MOAB_DOES_NOT_EXIST)
       results.report_results
+      preserved_copy.save!
       return
```

Comment:
Do we want an inline `if` for these saves? I remember this topic coming up in standup this morning...

Reply:
> I think .save! will only do its thing if .changed? returns true. I just experimented in irb and that's what I got.

yup.  as i recall, that was actually why we checked the `changed?` flag, so we could `touch` if the object hadn't actually changed (thus forcing `updated_at` to get updated).